docs/datamodel/functions.rst

```diff
@@ -19,11 +19,19 @@ A function may be defined in EdgeDB Schema using the ``function`` declaration:
 
     # where <argspec> is:
 
-    [ <argkind> ] $<argname>: [ <typequal> ] <argtype> [ = <default> ]
+    [ <argkind> ] <argname>: [ <typequal> ] <argtype> [ = <default> ]
+
+    # <argkind> is:
+
+    [ { variadic | named only } ]
+
+    # <typequal> is:
+
+    [ { set of | optional } ]
```

Comment:
Do support 'SINGLETON' there?  It's the default, and I'm wondering if our parser understands it.  If not -- it's ok, if yay -- need to document that.

Reply:
No, we don't support an explicit `SINGLETON` there. In fact, `SINGLETON` is no longer a keyword at all. We have another keyword `SINGLE` to be used specifically with links/properties.

Reply:
Alright then.